src/types/index.ts

```diff
@@ -0,0 +1,7 @@
+export type ErrorInfo = {
+    errorMessage: string;
+};
+
```

Comment:
Do you mean just have them be string arrays on each page?

Reply:
Yeah kind of - well objects with the `errorMessage: string;` declaration inside.
e.g. In ErrorSummary.tsx we could have 
`export type ErrorSummaryInfo = {
    errors: { errorMessage: string }[];
    errorHref?: string;
};`

As far as I can remember, we got rid of the types files we had previously for this reason.

Reply:
Going to keep it for now as it seems to break a load of stuff if we change that - i think @ljones92 said he wanted to have a bunch of interfaces rather than types anyway so I think we have some refactoring to do regardless. Definitely a fair point though, dont disagree.